scripts/tests/test_validate_docstrings.py

```diff
@@ -399,11 +398,10 @@ def test_exit_status_for_main(self, monkeypatch) -> None:
             func_name="docstring1",
             prefix=None,
             output_format="default",
-            errors=[],
             ignore_deprecated=False,
             ignore_errors=None,
         )
-        assert exit_status == 0
+        assert exit_status == 3
```

Comment:
When calling the script for a single function, until now it always returned an exit status of 0 even when there were errors. We don't really check this status anywhere right now, but I think it makes more sense that it also returns the number of errors, as we do when we call the script for all functions.

This is why I the exit status needs to be changed here.